src/classes/BDE_BatchEntry_CTRL.cls

```diff
@@ -451,5 +451,36 @@ public with sharing class BDE_BatchEntry_CTRL {
         pageRef.getParameters().put('obj', objName);
         pageRef.setRedirect(true);
         return pageRef;         
-    }             
+    }
+
+    public String getNamespace() {
```

Comment:
There's a few usages of this type of function in a few LWCs where we return `'c'` instead of `null`, I think this is a good candidate to make into a function in UTIL_Namespace.

Reply:
Done!